app/controllers/hours_controller.rb

```diff
@@ -21,6 +21,7 @@ def update
   def edit
     super
     resource
+    @last_hour_logged = Hour.last
```

Comment:
A problem arises when multiple users add entries at the same time, you might end up with a form prefilled with your colleagues project. I think you might want to add a reference to the created entry on the redirect and use that to prefill the form....

Reply:
Thanks, @tarzan I will do that.

Reply:
Hi @tarzan, concerning this task, instead of passing in the Hour.last  or Mileage.last, I decided to pass in current_user.hours.last which is last hour entry by that user. So this ensures that the hours displayed is that of the current user when the user wants to fill up the entry or mileage form. Adding a reference to the created entry on the redirect may not give us the behaviour we want because that value is only available when a user creates the entry when the user signout and returns back later, the user may not see the values prefilled.
Those
 this sound good?